src/screens/CreateEventPage.js

```diff
@@ -28,20 +27,20 @@ class CreateEventPage extends Component {
 
```

Comment:
What is `this.baseState = this.state` doing?

Reply:
resets the state of the fields to empty, in case someone hits the back button.